junit-test/shared/src/test/scala/org/scalajs/junit/MethodNameDecodeTest.scala

```diff
@@ -0,0 +1,35 @@
+package org.scalajs.junit
+
+import org.junit.Test
+import org.scalajs.junit.utils.JUnitTest
+
+class MethodNameDecodeTest {
+  @Test def `abcd ∆ƒ 😀 * #&$`(): Unit = ()
```

Comment:
I actually recommend using `\uD83D\uDE00` instead of the actual astral character. IDEs don't like astral characters. Since `\u` processing happens "pre-lexer" in Scala, it's fine to do so.